### DISCUSSION TITLE
Build production Error

### DISCUSSION THREAD

**Discussion by EmanueleGuidotti** [payloadcms/payload#7319] [Q&A]
### Payload Version

"payload": "^2.0.0"

### Node Version

node:18.8-alpine

### Next.js Version

"next": "13.5.2"

### Describe the Bug

When I run `yarn build` I got this error: 

```
yarn run v1.22.22
$ cross-env PAYLOAD_CONFIG_PATH=src/payload/payload.config.ts payload build
/Users/emanuele/Sites/arkage_website/node_modules/@swc/core/index.js:255

        throw new Error("Bindings not found");
```

### Reproduction Steps

Clone the payload repo with website template, connect to postgresql, run `yarn && yarn dev`, run `yarn build`

**Comment by jmikrut** [comment]
How did you install your dependencies? 

If NPM, you may need to use `npm i --legacy-peer-deps`.

In general, we suggest PNPM though. Will you give that a shot to see if that helps?

**Comment by EmanueleGuidotti** [comment]
I used yarn to install my dependencies

**Comment by EmanueleGuidotti** [comment]
I found the reason of the error, i was using node version 16 and the system was setted for node version 18. 
And now when I try to build the app I got this error instead:

```
[12:41:39] ERROR (payload): Error: cannot connect to Postgres. Details: getaddrinfo ENOTFOUND postgres
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This is my Database URI: 
```
DATABASE_URI="postgres://${DB_USER}:${DB_PASSWORD}@postgres:5432/${DB_NAME}"
```

**Comment by jmikrut** [comment]
Hey @EmanueleGuidotti — that is actually an issue with your database connection string or DB itself, not necessarily with Payload. You should try and use your exact connection string with a Postgres GUI tool like Beekeeper Studio to see if you can connect to your DB using the connection string provided to Payload. If you can, then you know the issue is related to the way you are passing the string to Payload. And if you can't, then you need to correct the connection string. 

I will convert this to a discussion as this is not an issue with Payload itself, but we're happy to continue to help!

**Reply by EmanueleGuidotti** [reply]
I'm using a docker container for postgresql. Maybe this is the problem?
